src/main/java/vazkii/pillar/StructureLoader.java

```diff
@@ -126,5 +127,41 @@ public static String getStructureNBTLocation(String jsonFileName) {
 	public static String jsonifySchema(StructureSchema schema) {
 		return gson.toJson(schema);
 	}
-	
+
+	public static StructureSchema getDefaultSchema() {
```

Comment:
I mean the end user. Though, in hindsight, the generatorType can be changed to have NONE, since that was added after the template.